playbooks/roles/csi-driver-config/README.md

```diff
@@ -13,6 +13,9 @@ Role Variables
 
 | Variable                       | Required | Default | Comments                                                      |
 |--------------------------------|----------|---------|---------------------------------------------------------------|
+| csidir                         | no       | ~/ibm-powervs-block-csi-driver | Place for csi config generation.       |
+| secret_template_url            | no       | `https://raw.githubusercontent.com/kubernetes-sigs/ibm-powervs-block-csi-driver/main/deploy/kubernetes/secret.yaml`   | Set to csi-driver secret file template url. |
```

Comment:
Instead for always downloading the simple secret from the internet, its good to maintain this template locally and use it